pyproject.toml

```diff
@@ -82,3 +83,14 @@ remove-all-unused-imports = true
 
 [tool.flit.module]
 name = "circuit_knitting_toolbox"
+
+[tool.ruff]
```

Comment:
Forgive my ignorance, so ruff operates through the build metafile rather than its own ini file?

Reply:
It [looks like](https://github.com/charliermarsh/ruff#configuration) we have the option of using `ruff.toml` or `.ruff.toml` if we so choose, instead of `pyproject.toml`.  I kind of like having all the configuration in `pyproject.toml`, but I think Pedro holds the opposite opinion, and I believe he holds it pretty strongly.  In his projects, typically only `black` is configured in `pyproject.toml`, as it doesn't support any other configuration file.  (I think he even filed an issue about this, and it was promptly closed.)  So yes, perhaps `.ruff.toml` is going to be the most clear compromise.

Reply:
I updated this in 746bcddd0beb883090c9c3bd1a491910f2e5225d.

Reply:
Great, thanks